redbot/cogs/customcom/customcom.py

```diff
@@ -227,6 +227,8 @@ def __init__(self, bot):
             await ctx.send(_("There already exists a bot command with the same name."))
             return
         responses = await self.commandobj.get_responses(ctx=ctx)
+        if not responses:
+            return await ctx.send(_("Custom command process cancelled."))
```

Comment:
It's not required here, but since we're trying to make our code type-safe, it would be better to not return message object:
```suggestion
            await ctx.send(_("Custom command process cancelled."))
            return
```